libs/react-components/src/lib/components/Alert/Alert.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import Alert from './index';
+import IconButton from '../IconButton';
+
+const meta = {
+  title: 'Components/Alert',
+  component: Alert,
+  // This component will have an automatically generated Autodocs entry: https://storybook.js.org/docs/writing-docs/autodocs
+  tags: ['autodocs'],
+  parameters: {
+    // More on how to position stories at: https://storybook.js.org/docs/configure/story-layout
+    layout: 'fullscreen',
+  },
+  args: {
+    icon: 'info',
+    titleText: 'Alert title',
+    descriptionText: 'Alert description',
+    state: 'active',
+  },
+} satisfies Meta<typeof Alert>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Basic: Story = {};
+
+export const ActionElements: Story = {
+  render: (args) => {
+    return (
+      <Alert {...args}>
+        <div slot="action-items">
+          <IconButton icon="close" />
```

Comment:
can we do without the div? it works but its simpler to just add the slot name to the single item here.

```suggestion
          <IconButton slot="action-items" icon="close" />
```